packages/strapi-plugin-users-permissions/services/UsersPermissions.js

```diff
@@ -231,4 +231,97 @@ module.exports = {
       strapi.log.error(err);
     }
   },
+
+  syncSchema: (cb) => {
+    const Model = strapi.plugins['users-permissions'].models.user;
+
+    if (Model.orm === 'bookshelf') {
+      const tableName = Model.collectionName;
+
+      let queu = new Promise((resolve, reject) => {
```

Comment:
Why do you need to define a variable which is empty because you're returning nothing?